substrate/runtime/council/src/lib.rs

```diff
@@ -291,10 +293,13 @@ impl<T: Trait> Module<T> {
 			voters
 		);
 		if valid {
-			<staking::Module<T>>::transfer_reserved_balance(&who, aux.ref_into(), Self::voting_bond())
+			// This only fails if `who` doesn't exist, which is clearly must do since its the aux.
+			// Still, it's no more harmful to propagate any error at this point.
+			<staking::Module<T>>::transfer_reserved(&who, aux.ref_into(), Self::voting_bond())?;
```

Comment:
perhaps separate functions for transferring when fully resolved and potentially unresolved would be useful?